### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by jeraldtsy**
For CSV files import/export feature and alternate choices

### CODE REVIEW

**Comment by eugenefdw** [comment]
Should be merged to v1.2 branch, not master

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -253,10 +253,21 @@ This section describes some noteworthy details on how certain features are imple
...
 
-* `import user` - Imports the selected CSV file and load the current progress of the user.
-* `export user` - Exports the current progress of the user into a CSV file.
+* 'import feature' - Imports the selected CSV file and load the current progress of the user.
+* `export feature` - Exports the current progress of the user into a CSV file.
+
+NOTE: These functions are automatically performed on opening/closing the Session.
+
+==== Design Considerations
+* ** Alternative 1(Current choice): Automate import/exporting when session is open/close
```
Comment (lallanachang) [inline docs/DeveloperGuide.adoc]: import/exporting should be in same tense here

File: docs/team/jeraldtsy.adoc [hunk 2]
```diff
@@ -3,23 +3,24 @@
...
 
 == Summary of contributions
 
-* *Major enhancement*: added *the ability to undo/redo previous commands*
-** What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-** Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-** Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-** Credits: _{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}_
+* *Major enhancement*: User features
+** What it does: User features manages the user progress and import and export their progress into a CSV file accordingly. The feature is automated whenever the program is open(Import) or close(export)
```
Comment (lallanachang) [inline docs/team/jeraldtsy.adoc]: "User features manage...."

File: src/main/java/seedu/address/model/user/CardSrsData.java [hunk 3]
```diff
@@ -6,7 +6,7 @@
 /**
  * Represents a Card which contains the data and hashcode that can be called by session
  */
-public class CardSrsData {
+public class CardSrsData extends User {
```
Comment (lallanachang) [inline src/main/java/seedu/address/model/user/CardSrsData.java]: May implement a single method to return the hashmap that contains cardSrsData as well.

File: src/main/java/seedu/address/model/user/User.java [hunk 4]
```diff
@@ -54,4 +57,13 @@ public void setName(String name) {
     public List<CardSrsData> getCardData() {
         return cardData;
     }
+
+    /**Add user
+     *
```
Comment (lallanachang) [inline src/main/java/seedu/address/model/user/User.java]: The doc format here seems need to be adjusted

Comment (lallanachang) [inline src/main/java/seedu/address/model/user/User.java]: Consider read the value and form a hashmap containing srscarddata and its hashcode for later use in model session generation.

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (jeraldtsy) [reply docs/DeveloperGuide.adoc]: Done, resolved

File: docs/team/jeraldtsy.adoc (hunk 2 above)
Comment (jeraldtsy) [reply docs/team/jeraldtsy.adoc]: Done, resolved

**Pull request merged**